packages/dx-react-grid/src/grid.jsx

```diff
@@ -32,19 +32,19 @@ Grid.propTypes = {
   getRowId: PropTypes.func,
   getCellValue: PropTypes.func,
   columns: PropTypes.array.isRequired,
-  rootTemplate: PropTypes.func.isRequired,
-  headerPlaceholderTemplate: PropTypes.func,
-  footerPlaceholderTemplate: PropTypes.func,
+  rootComponent: PropTypes.func.isRequired,
```

Comment:
Is `PropTypes.func` correct for this property?

Reply:
Yes. This is the way React components can be validated